scripts/create-new-package.bash

```diff
@@ -68,30 +70,77 @@ case "$META" in
   echo -e "${TERMINAL_COLOR_USER_NOTICE}Package '$PKG_NAME' will be created!${TERMINAL_COLOR_NC}"
 esac
 
-user_decision "Do you want to enter name and email address of the maintainer? If not, data from git configuration will be used."
-if [[ " ${positive_answers[*]} " =~ " ${user_answer} " ]]; then
-  choice="y"
-else
-  choice="n"
+
+# MAINTAINER_NAME and MAINTAINER_EMAIL options for a multiple choice
+maintainer_info_user_input_option="user input"
+maintainer_info_options=("$maintainer_info_user_input_option")
+
+global_git_name=`git config --global user.name`
+global_git_email=`git config --global user.email`
+if [ -n "$global_git_name" ] && [ -n "$global_git_email" ]; then
+  maintainer_info_global_git_option="global git: $global_git_name, $global_git_email"
```

Comment:
Git usually uses the following syntax for the authors: "Author Name <author.name@stuff.blup>"

The same should be done for the local things.

P.S. you can directly accept this change here in GitHub.

```suggestion
  maintainer_info_global_git_option="global git: $global_git_name <$global_git_email>"
```